api/authentication.py

```diff
@@ -29,7 +29,7 @@ def get_cache_key(token):
 
 def introspect_token(token: str) -> dict[str, Any]:
     try:
-        fxa_resp = requests.post(INTROSPECT_TOKEN_URL, json={"token": token})
+        fxa_resp = requests.post(INTROSPECT_TOKEN_URL, json={"token": token}, timeout=1)
```

Comment:
I'm not sure if 1 second is the right value for FxA. Maybe move it to a setting that defaults to 1? That way we can adjust after deploying without a code change.

Reply:
Updated to use `settings.FXA_REQUESTS_TIMEOUT_SECONDS`